diff -up dolphin-21.07.80/src/main.cpp.1~ dolphin-21.07.80/src/main.cpp
--- dolphin-21.07.80/src/main.cpp.1~	2021-07-17 03:30:22.104570478 +0200
+++ dolphin-21.07.80/src/main.cpp	2021-07-17 03:31:01.852154051 +0200
@@ -39,19 +39,6 @@
 
 int main(int argc, char **argv)
 {
-#ifndef Q_OS_WIN
-    // Prohibit using sudo or kdesu (but allow using the root user directly)
-    if (getuid() == 0) {
-        if (!qEnvironmentVariableIsEmpty("SUDO_USER")) {
-            std::cout << "Executing Dolphin with sudo is not possible due to unfixable security vulnerabilities." << std::endl;
-            return EXIT_FAILURE;
-        } else if (!qEnvironmentVariableIsEmpty("KDESU_USER")) {
-            std::cout << "Executing Dolphin with kdesu is not possible due to unfixable security vulnerabilities." << std::endl;
-            return EXIT_FAILURE;
-        }
-    }
-#endif
-
     /**
      * enable high dpi support
      */
