diff -up dolphin-20.03.90/src/settings/general/confirmationssettingspage.cpp.omv~ dolphin-20.03.90/src/settings/general/confirmationssettingspage.cpp
--- dolphin-20.03.90/src/settings/general/confirmationssettingspage.cpp.omv~	2020-04-10 15:05:35.806235230 +0200
+++ dolphin-20.03.90/src/settings/general/confirmationssettingspage.cpp	2020-04-10 15:05:43.210322971 +0200
@@ -110,7 +110,7 @@ ConfirmationsSettingsPage::Confirmations
     connect(m_confirmMoveToTrash, &QCheckBox::toggled, this, &ConfirmationsSettingsPage::changed);
     connect(m_confirmEmptyTrash, &QCheckBox::toggled, this, &ConfirmationsSettingsPage::changed);
     connect(m_confirmDelete, &QCheckBox::toggled, this, &ConfirmationsSettingsPage::changed);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(m_confirmScriptExecution, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &ConfirmationsSettingsPage::changed);
 #else
     connect(m_confirmScriptExecution, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &ConfirmationsSettingsPage::changed);
diff -up dolphin-20.03.90/src/settings/viewmodes/viewsettingstab.cpp.omv~ dolphin-20.03.90/src/settings/viewmodes/viewsettingstab.cpp
--- dolphin-20.03.90/src/settings/viewmodes/viewsettingstab.cpp.omv~	2020-04-10 15:05:03.905857187 +0200
+++ dolphin-20.03.90/src/settings/viewmodes/viewsettingstab.cpp	2020-04-10 15:05:12.019953346 +0200
@@ -120,19 +120,19 @@ ViewSettingsTab::ViewSettingsTab(Mode mo
 
     switch (m_mode) {
     case IconsMode:
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
         connect(m_widthBox, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &ViewSettingsTab::changed);
 #else
         connect(m_widthBox, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &ViewSettingsTab::changed);
 #endif
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
         connect(m_maxLinesBox, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &ViewSettingsTab::changed);
 #else
         connect(m_maxLinesBox, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &ViewSettingsTab::changed);
 #endif
         break;
     case CompactMode:
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
         connect(m_widthBox, QOverload<int>::of(&QComboBox::currentIndexChanged), this, &ViewSettingsTab::changed);
 #else
         connect(m_widthBox, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged), this, &ViewSettingsTab::changed);
diff -up dolphin-20.03.90/src/settings/viewpropertiesdialog.cpp.omv~ dolphin-20.03.90/src/settings/viewpropertiesdialog.cpp
--- dolphin-20.03.90/src/settings/viewpropertiesdialog.cpp.omv~	2020-04-10 15:03:15.055567005 +0200
+++ dolphin-20.03.90/src/settings/viewpropertiesdialog.cpp	2020-04-10 15:03:25.181687043 +0200
@@ -163,19 +163,19 @@ ViewPropertiesDialog::ViewPropertiesDial
     layout->addRow(QString(), m_showInGroups);
     layout->addRow(QString(), m_showHiddenFiles);
 
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(m_viewMode, QOverload<int>::of(&QComboBox::currentIndexChanged),
 #else
     connect(m_viewMode, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged),
 #endif
             this, &ViewPropertiesDialog::slotViewModeChanged);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(m_sorting, QOverload<int>::of(&QComboBox::currentIndexChanged),
 #else
     connect(m_sorting, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged),
 #endif
             this, &ViewPropertiesDialog::slotSortingChanged);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
+#if QT_VERSION <= QT_VERSION_CHECK(5, 15, 0)
     connect(m_sortOrder, QOverload<int>::of(&QComboBox::currentIndexChanged),
 #else
     connect(m_sortOrder, QOverload<int, const QString &>::of(&QComboBox::currentIndexChanged),
